src/tenant/tenant.resolver.ts

```diff
@@ -45,4 +46,35 @@ export default class TenantResolver {
 
     return tenant;
   }
+
+  @Mutation(() => Boolean)
+  async markRentAsLate(@Arg('id', () => Int) id: number): Promise<boolean> {
+    const t = await Tenant.findOne({where: {id}});
+    if (t) {
+      const l = await Lease.find({where: {tenantId: id}});
+      if (l.length === 0) {
+        throw new Error("Tenant doesn't have any leases");
+      }
+
+      t.isRentLate = true;
+      await t.save();
+    }
+
+    return true;
```

Comment:
Shouldn't we return `true` only when the update occurs?

```suggestion
      await t.save();
      return true;
    }

    return false;
```